pkgs/os-specific/linux/joycond/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libevdev, udev }:
+
+stdenv.mkDerivation rec {
+  pname = "joycond-unstable";
+  version = "2020-11-05";
+
+  src = fetchFromGitHub {
+    owner = "DanielOgorchock";
+    repo = "joycond";
+    rev = "2990fe2b156697e7eca6f32be5a2d22ed26f6386";
+    sha256 = "0zgxb92ybq1pd2x7ncgk5knskhhzrkh3g35sfflwm4d3rn9g3ca5";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ libevdev udev ];
+
+  # CMake has hardcoded install paths
+  installPhase = ''
+    mkdir -p $out/{bin,etc/systemd/system,lib/{udev/rules.d,modules-load.d}}
+
+    cp ./joycond $out/bin
+    cp $src/udev/{89,72}-joycond.rules $out/lib/udev/rules.d
```

Comment:
I believe you should copy the files to `$out/etc/udev/rules.d` instead of `$out/lib/udev/rules.d`. Then the udev rules can be installed just by adding `pkgs.joycond` to `services.udev.packages`.

Reply:
Thanks! I'm not too familiar with udev rules, so i really appreciate this feedback.